### PR TITLE
formatting updates in the sample output

### PR DESCRIPTION
**Pull request by alexlancaster**
This is great @rsingle - thanks!  Quick question: is this what the default output will now look like? (I haven't checked it recently)

### CODE REVIEW

**Comment by rsingle** [comment]
> Quick question: is this what the default output will now
> look like? (I haven't checked it recently)
Yes, definitely for the "+" vs. ":" in the HWP results and "~" vs. ":" in the haplo frequency results.  The USAFEL-UchiTelle .pop files in data/samples/ have the old allele names, so the alleles will not be colon-delimited unless we change the sample data.

**Comment by alexlancaster** [comment]
> Yes, definitely for the "+" vs. ":" in the HWP results and "~" vs. ":" in the haplo frequency results.  The USAFEL-UchiTelle .pop files in data/samples/ have the old allele names, so the alleles will not be colon-delimited unless we change the sample data.

I wonder if we shouldn't maybe also add a note explaining that the alleles are in that format.  Since all the previous examples in the usage in the previous chapter (http://pypop.org/docs/guide-chapter-usage.html) all use the "old" nomenclature, it might throw the reader a bit to see alleles with that format.  It looks like the `.ini` file doesn't need to be modified to use the `:` colons, right?  Maybe we could also provide a link to an example `.pop` file from the unit tests?  e.g. https://github.com/alexlancaster/pypop/blob/main/tests/data/Test_Allele_Colon_HardyWeinberg.pop.  What do you think?

**Comment by alexlancaster** [comment]
> I wonder if we shouldn't maybe also add a note explaining that the alleles are in that format. Since all the previous examples in the usage in the previous chapter (http://pypop.org/docs/guide-chapter-usage.html) all use the "old" nomenclature, it might throw the reader a bit to see alleles with that format. It looks like the `.ini` file doesn't need to be modified to use the `:` colons, right? Maybe we could also provide a link to an example `.pop` file from the unit tests? e.g. https://github.com/alexlancaster/pypop/blob/main/tests/data/Test_Allele_Colon_HardyWeinberg.pop. What do you think?

@rsingle I added in a note at the top of the chapter that says:

> In  contrast to the sample files shown in [Getting started with PyPop](https://alexlancaster.github.io/beta.pypop.org/docs/guide-chapter-usage.html), the example outputs below were all generated from an input .pop file that encodes the alleles using the [new HLA nomenclature](https://hla.alleles.org/nomenclature/naming.html).

If you don't have any objections, I'll merge this to `main` very soon.

File: website/docs/guide-chapter-usage.rst
Comment (alexlancaster) [inline website/docs/guide-chapter-usage.rst:1]: Looks good!

**Pull request merged**